scripts/lib-node.sh

```diff
@@ -8,7 +8,7 @@ then if test -x ./scripts/genesis.sh
      fi
 fi
 
-genesis_hash=`cat ${genesis_root}/GENHASH`
+genesis_hash="cat ${genesis_root}/GENHASH"
```

Comment:
that will not work! try it in a shell and you will see that the $genesis_hash will contain a string: "cat configuration/Something/GENHASH", but not the genesis' hash.

Reply:
the meaning of backticks is: run the command and get its output
whereas double quotes are only to enclose strings

Reply:
@CodiePP Hi, after branch checkout I ran those scripts again (`genesis.sh` and then `get-default-key-address.sh`)  and everything works fine... **HOWEVER** you are right - this line is still wrong. Thanks for pointing this.

Leaving it like it was before change: `` `cat ${genesis_root}/GENHASH` `` will not work for directory with white spaces in name but I think the change below should work for all cases:

`` genesis_hash=`cat "${genesis_root}/GENHASH"` ``

Reply:
👍